spec/discourse_code_review/lib/state/github_repo_categories_spec.rb

```diff
@@ -16,9 +17,24 @@
     c.custom_fields[DiscourseCodeReview::State::GithubRepoCategories::GITHUB_REPO_NAME] = "repo-name"
     c.save_custom_fields
     expect {
-      c = described_class.ensure_category(repo_name: "repo-name")
+      c = described_class.ensure_category(repo_name: "repo-name", repo_id: 242)
```

Comment:
Since `repo_id` is an optional parameter I think we should add a test that makes sure things work when it's not present.

Reply:
Added a test.